gst/nnstreamer/ml_agent.h

```diff
@@ -21,9 +21,9 @@
 /**
  * @brief Parse the given URI into the valid file path string
  */
-const gchar *mlagent_parse_uri_string (const GValue * val);
+gchar *mlagent_parse_uri_string (const GValue * val);
```

Comment:
Add a note for the returned pointer; whether the caller should free or not.

Reply:
Updated it. Thanks.